sql/src/main/java/io/crate/planner/node/management/ExplainPlan.java

```diff
@@ -167,18 +172,80 @@ private TransportCollectProfileOperation getRemoteCollectOperation(DependencyCar
         return new TransportCollectProfileOperation(nodeAction, jobId);
     }
 
-    private Row buildResponse(Map<String, Object> apeTimings, Map<String, Map<String, Object>> nodeTimings) {
+    private Row buildResponse(Map<String, Object> apeTimings,
+                              Map<String, Map<String, Object>> timingsByNodeId,
+                              NodeOperationTree operationTree) {
         MapBuilder<String, Object> mapBuilder = MapBuilder.newMapBuilder();
         apeTimings.forEach(mapBuilder::put);
 
+        Map<String, Object> phasesTimings = extractPhasesTimingsFrom(timingsByNodeId, operationTree);
```

Comment:
Maybe we could inline a comment that shows the  structure transformation based on an example? Following `Map -> Map` transformations is a bit abstract without a picture of the data